.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please pick a meaningful title for your pull request using sentence case.
+
+Do not overuse punctuation in the title like `(chore):`. If it is helpful feel free to start with a project name, though, like `ProjectX: Implement some feature`.
+
+# Title instructions above.
+
+Enter a succinct description of what is achieved by the PR. Ideally describe why the change is being made.
+
+Bullet points like
+
+- Implements aspect X
```

Comment:
Ha, really good question. I changed the title one for now. I don't find that particularly important.